Config/CommonConfiguration.swift

```diff
@@ -94,7 +94,10 @@ class CommonConfiguration: NSObject, Configurable {
         sdkOptions.enableNewClientInformationFeature = RiotSettings.shared.enableClientInformationFeature
         
         if sdkOptions.isCryptoSDKAvailable {
-            let isEnabled = RiotSettings.shared.enableCryptoSDK
+            // Tchap : force to not use the new Rust based Crypto SDK
+//            let isEnabled = RiotSettings.shared.enableCryptoSDK
+            let isEnabled = false
```

Comment:
The value is false by default. Why do you need to force this value ?

Reply:
@NicolasBuquet my comment on this code was a remark. You should keep the code from Element here
This is important to track RiotSettings.shared.enableCryptoSDK in your next rebase (to detect when it would be true by default). Currently it is false by default

Reply:
I reverted the commit.
I saw the default value was false. But I didn't know if this property was changed to true by any way in another place of the sdk. So, to be sure, I preferred set it to false when using it in Tchap.